_delphi_utils_python/tests/validator/test_datafetcher.py

```diff
@@ -21,25 +21,41 @@ def test_make_date_filter(self):
         assert not date_filter(FILENAME_REGEX.match("20200620_a_b.csv"))
         assert not date_filter(FILENAME_REGEX.match("202006_a_b.csv"))
 
-    # pylint: disable=fixme
-    # TODO: mock out the advanced meta endpoint /covidcast/meta as well
-    # https://github.com/cmu-delphi/covidcast-indicators/issues/1456
+    # Solution from https://stackoverflow.com/questions/15753390/
+    #how-can-i-mock-requests-and-the-response
```

Comment:
Yeah, linking to where you found a solution is fine, especially if the approach is tricky or complex or non-standard in some way. You can also link to explanations of bugs for larger or more complex bugs.